shell/platform/fuchsia/flutter/flatland_external_view_embedder.cc

```diff
@@ -36,6 +36,20 @@ FlatlandExternalViewEmbedder::FlatlandExternalViewEmbedder(
   root_transform_id_ = flatland_->NextTransformId();
   flatland_->flatland()->CreateTransform(root_transform_id_);
   flatland_->flatland()->SetRootTransform(root_transform_id_);
+
+  if (intercept_all_input) {
+    input_interceptor_node_ = flatland_->NextTransformId();
```

Comment:
Can we call it input_interceptor_transform_ instead? Node is a gfx specific resource.

Reply:
Done